chihacknight/decarbonize-my-state
# Meta and head updates

## Overview

This PR adds more complete meta and head tags as well as fixes the missing `lang=en` tag needed on the base `html` element.

Closes #81 & #83 and parts of #73 

### Demo
<img width="571" alt="Screen Shot 2022-05-11 at 8 24 24 AM" src="https://user-images.githubusercontent.com/10041771/167860705-6c425f8d-3619-41d0-b550-e0c57f852a1f.png">

<img width="571" alt="Screen Shot 2022-05-11 at 8 24 16 AM" src="https://user-images.githubusercontent.com/10041771/167860720-6fbf675a-c8f7-4114-81cd-8ac3557a6733.png">


![image](https://user-images.githubusercontent.com/10041771/167860677-0e1e5c58-f73c-42e2-ab45-9cf00ef35245.png)


### Notes

Helmet was already installed!

## Testing Instructions

- Go to page, view head element
- Check branch preview with [Twitter's card validator](https://cards-dev.twitter.com/validator) and [Facebook's sharing debugger](https://developers.facebook.com/tools/debug/)
